nixos/modules/config/system-environment.nix

```diff
@@ -23,7 +23,7 @@ in
         strings.  The latter is concatenated, interspersed with colon
         characters.
       '';
-      type = types.attrsOf (types.loeOf types.str);
+      type = with types; attrsOf (either str (listOf str));
```

Comment:
@ericsagnes This change prevents environment variables from being set in multiple modules. That prevents, e.g. `LD_LIBRARY_PATH` being set for OpenGL and SANE simultaneously.